### PR TITLE
Add banner to all jobs regarding Jenkins shutdown August 2022

### PR DESCRIPTION
**Pull request by DennisPeriquet**
Hopefully this message will remind anyone using the UI to plan ahead.

This has been rolled out Apr 21, 2022 ~6:15pm EST.

### CODE REVIEW

**Comment by jupierce** [comment]
@DennisPeriquet - I don't want to bike shed you as this is already above and beyond. But as a user of this system, my eye is likely trained to completely ignore this yellow banner. Could we add a second, independent div, red, larger font?

**Comment by dperique** [comment]
Good idea. Let me see about how to do that.

I looked at that message and it didn't quite jump out at me either.

File: sjb/generate.py [hunk 1]
```diff
@@ -258,6 +258,7 @@ def parse_action(action):
     debug("[INFO] Marking this as a {} job for the {} repo".format(action, target_repo))
 
 DEFAULT_DESCRIPTION = "<div style=\"font-size: 32px; line-height: 1.5em; background-color: yellow; padding: 5px;\">" + \
```
Comment (jupierce) [inline sjb/generate.py:260]:
```suggestion
DEFAULT_DESCRIPTION = "<div style=\"font-size: 32px; line-height: 1.5em; background-color: red; padding: 5px;\">*** THIS SERVER WILL BE SHUTDOWN AUGUST 2022 ***</div><div style=\"font-size: 16px; line-height: 1.5em; background-color: yellow; padding: 5px;\">" + \
```

Comment (jupierce) [reply sjb/generate.py:260]: Mostly the same, just break out into another div.

Comment (DennisPeriquet) [reply sjb/generate.py]: thanks
Screen Shot 2022-04-21 at 6 38 30 PM

**Review by jupierce — APPROVED** [review]

**Comment by jupierce** [comment]
Thanks, @DennisPeriquet!

**Pull request merged**